recipes/elyra/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "elyra" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/0b/0d/7355f28392a40bca18a373c2f6c55306a3f345ea257afa6e3a074a6e9785/elyra-{{ version }}-py3-none-any.whl
+  sha256: 3c713cc33cb309caaa41176a33e187012b5092ec611b1ad063e9ea65a743806b
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  script: "{{ PYTHON }} -m pip install elyra-{{ version }}-py3-none-any.whl --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+  run:
+    - python >=3.6
+    - jupyter_core >=4.0,<5.0
+    - kfp ==0.5.1
+    - urllib3 ==1.24.2
+    - kfp-notebook >=0.11.0,<0.12.0
+    - minio >=5.0.7
+    - nbdime >=2.0.0
+    - jupyterlab >=2.0.0,<3.0.0
+    - jupyterlab-git >=0.20.0
+    - nbconvert >=5.6.1
+    - notebook >=6.0.3
+    - traitlets >=4.3.2
+    - jsonschema >=3.2.0
+    - requests >=2.9.1,<3.0
+    - entrypoints >=0.3
+    - rfc3986-validator >=0.1.1
+    - autopep8
+    - jinja2
+
+test:
+  imports:
+    - elyra
```

can we use ``pip check`` in the tests as well please?
```suggestion
    - elyra
  requires:
    - pip
  commands:
    - pip check
```